.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
```

Comment:
This seems redundant.. any editor worth is salt already knows that Makefiles need tab indentation.

Otherwise lgtm

Reply:
Fair enough. Updated.